projects_contribution.md

```diff
@@ -0,0 +1,16 @@
+# Galaxy Training Network
```

Comment:
I do not understand the purpose of this page, compared to the one before.

Reply:
The "Galaxy Training Network" is like the "Galaxy Tools" (the one below). So on this page, we have a list of topic related GTN. But all that we change since we are opting for "Past Gmentees".

Reply:
I got that. I just do not understand why there is an extra page for that.